src/components/Box/__stories__/Box.stories.mdx

```diff
@@ -0,0 +1,491 @@
+import Box from "../Box";
+import {ArgsTable, Story, Canvas, Meta} from "@storybook/addon-docs";
+import {createStoryMetaSettings} from "../../../storybook/functions/create-component-story";
+import styles from "./Box.stories.module.scss";
+
+export const metaSettings = createStoryMetaSettings({
+    component: Box,
+    enumPropNamesArray: [
+        "children",
+        "disabled",
+        "border",
+        "borderTop",
+        "borderEnd",
+        "borderBottom",
+        "borderStart",
+        "borderColor",
+        "rounded",
+        "roundedTop",
+        "roundedEnd",
+        "roundedBottom",
+        "roundedStart",
+        "shadow",
+        "margin",
+        "marginX",
+        "marginY",
+        "marginTop",
+        "marginEnd",
+        "marginStart",
+        "padding",
+        "paddingX",
+        "paddingY",
+        "paddingTop",
+        "paddingEnd",
+        "paddingBottom",
+        "paddingStart",
+        "backgroundColor",
+        "text",
```

Comment:
text? why we need children and also text? the all idea is that the box can contain anything no?

Reply:
ok this is text color... let's change the name to text color please it's confusing a bit

Reply:
ok

Reply:
done